app/Filament/Admin/Widgets/CanaryWidget.php

```diff
@@ -2,31 +2,42 @@
 
 namespace App\Filament\Admin\Widgets;
 
-use Filament\Actions\CreateAction;
-use Filament\Widgets\Widget;
+use Filament\Forms\Components\Actions\Action;
+use Filament\Forms\Components\Placeholder;
+use Filament\Forms\Components\Section;
+use Filament\Forms\Form;
 
-class CanaryWidget extends Widget
+class CanaryWidget extends FormWidget
 {
-    protected static string $view = 'filament.admin.widgets.canary-widget';
-
-    protected static bool $isLazy = false;
-
     protected static ?int $sort = 1;
 
     public static function canView(): bool
     {
         return config('app.version') === 'canary';
     }
 
-    public function getViewData(): array
+    public function form(Form $form): Form
     {
-        return [
-            'actions' => [
-                CreateAction::make()
-                    ->label(trans('admin/dashboard.sections.intro-developers.button_issues'))
-                    ->icon('tabler-brand-github')
-                    ->url('https://github.com/pelican-dev/panel/issues', true),
-            ],
-        ];
+        return $form
+            ->schema([
+                Section::make(trans('admin/dashboard.sections.intro-developers.heading'))
+                    ->icon('tabler-code')
+                    ->iconColor('primary')
+                    ->collapsible()
+                    ->collapsed()
```

Comment:
```suggestion
                    ->persistCollapsed()
```
other then that LGTM